mat64/dense.go

```diff
@@ -257,7 +257,7 @@ func (m *Dense) SetRow(i int, src []float64) int {
 	return min(len(src), m.mat.Cols)
 }
 
-// RowView returns a Vector reflecting row i, backed by the matrix data.
+// RowView returns a Vector reflecting row i transposed, backed by the matrix data.
```

Comment:
While I'm a great fan of parenthetic writing, they don't read well in documentation.